test/plugins/json_test.lua

```diff
@@ -58,3 +58,21 @@ g.test_number_value_serialization = function()
     t.assert_equals(float_obs.value, 0.333, 'check float')
     t.assert_equals(int_obs.value, 10, 'check int')
 end
+
+g.test_cdata_ll_value_parses_to_number = function()
+    local gauge_ll = metrics.gauge('test_cdata_ll')
+    gauge_ll:set(-205LL)
```

Comment:
number64 values (cdata<int64_t> / cdata<uint64_t>) usually occurs when you working with very large (or very small negative) integral numbers. So I think that we should test those cases.

Reply:
Well, I've got ULL suffix in `51589ULL`. But I can increase the number

Reply:
Changed

Reply:
`51589ULL` is less than 10^14 (where we started to encode it into cdata) and less than 2^53, where we can loss precision below 1.0. We should check that, say `9007199254740992ULL` (2^53) and numbers around it are marshalled correctly.

You can consider this following illustration how integral numbers are marshalled in tarantool:

https://github.com/tarantool/conf/blob/80078f85423ffd54d812c11b47aa85729e9e3d87/conf/client/etcd/scalar_serializer.lua#L134-L152

Reply:
Looks like space bsize is always cdata
```
Tarantool 2.10.0-beta1-0-g7da4b1438
type 'help' for interactive help
tarantool> box.cfg{}
---
...
tarantool> box.space._space:bsize()
---
- 4151
...

tarantool> type(box.space._space:bsize())
---
- cdata
...
```

Reply:
(Tarantool 2.10.0-beta1-91-g16a484a30.)

```
tarantool> print(json.encode(box.space._space:bsize()))
7943
---
...
```

Do you see another behaviour of the json encoder on another tarantool version?

Reply:
Actually no, it is as you described. Looks like I was wrong. Thank you! I had reworked json export to be as simple as possible.

Reply:
Okay so.